UI.tsv

```diff
@@ -568,62 +568,62 @@ UI_20150317_000567	{@st41}Damage Top 100{/}
 UI_20150317_000568	{@st41}Money Money Top 100{/}
 UI_20150317_000569	{@st41}Level Up Top 100{/}
 UI_20150317_000570	{@st41b}Menu{/}
-UI_20150317_000571	{@st43}{s22}Enhance Item{/}
+UI_20150317_000571	{@st43}{s22}Refine Item{/}
 UI_20150317_000572	{@st41b}Selected Item
 UI_20150317_000573	{@st41b}Anvil
 UI_20150317_000574	{@st41}{s20}Anvil Hit Count :{@st43} %s
 UI_20150317_000575	{@st41} {s20}Price : %s
-UI_20150317_000576	{@st41b}Enhance{/}
-UI_20150317_000577	{@st59}Start Item Enhancement{/}
-UI_20150317_000578	{@st59}Close Enhancement{/}
-UI_20150317_000579	{@st41b} Would you like to enhance the {@sti9}×%s{/} selected items?{/}
-UI_20150317_000580	{memo X}{@st43}Enhance
-UI_20150317_000581	{@st44}Enhance Item{/}
-UI_20150317_000582	{s24}{b}{#000000}Enhancement Completed!
-UI_20150317_000583	{@st41}Enhance Again{/}
+UI_20150317_000576	{@st41b}Refine{/}
+UI_20150317_000577	{@st59}Start the Item Refinement{/}
+UI_20150317_000578	{@st59}Close Refinement{/}
+UI_20150317_000579	{@st41b} Would you like to refine the {@sti9}×%s{/} selected items?{/}
+UI_20150317_000580	{memo X}{@st43}Refine
+UI_20150317_000581	{@st44}Refine Item{/}
+UI_20150317_000582	{s24}{b}{#000000}Refinement Completed!
+UI_20150317_000583	{@st41}Refine Again{/}
```

Comment:
There is nothing wrong with the previous. 'Enhance' is more commonly used and known.